src/uwtools/tests/drivers/test_driver.py

```diff
@@ -3,9 +3,75 @@
 Tests for uwtools.drivers.driver module.
 """
 
+from importlib import resources
+from types import SimpleNamespace as ns
+from unittest.mock import patch
+
+import pytest
+from pytest import fixture
+
 from uwtools.drivers import driver
+from uwtools.drivers.driver import Driver
+
+
+@fixture
+def create_schema(tmp_path):
+    schema_file = tmp_path / "schema_file.jsonschema"
+    with open(schema_file, "w", encoding="utf-8") as f:
+        print("undefined_filter: '{{ 34 | not_a_filter }}'", file=f)
+    return schema_file
+
+
+@pytest.mark.usefixtures("create_schema")
+class Testinstance(Driver):
+    """
+    Test concrete class instantiation.
+    """
+
+    @property
+    def schema(self) -> str:
+        return create_schema
+
+    def requirements(self):
+        """
+        requirements.
+        """
```

Comment:
A common Python idiom for do-nothing functions/methods is to just use the statement `pass` as the body, e.g.
```
def requirements(self):
    pass
```
Slightly more concise, FWIW, if you like it. Thought I'd share.